core/trino-main/src/main/java/io/trino/operator/DirectExchangeClientStatus.java

```diff
@@ -113,6 +118,12 @@ public List<PageBufferClientStatus> getPageBufferClientStatuses()
         return pageBufferClientStatuses;
     }
 
+    @JsonProperty
+    public synchronized TDigestHistogram getRequestsDuration()
```

Comment:
remove synchronized

Reply:
I cannot. You have to make a copy of `requestsDuration` because it can be accessed (read) by multiple threads. 
Yes, reading of `TDigest` can modify the structure of `this`